client/chatbotWidget/src/components/HelpDeskWidget.tsx

```diff
@@ -39,10 +39,10 @@ const HelpDeskWidget: React.FC = () => {
         // Clear error message after 3 seconds
         setTimeout(() => {setError(null)}, 3000); //
       }
-      finally{
-        // Clearing out the messages to allow for new input
-        setMessage('');
-      }
+      
+      // Clearing out the messages to allow for new input
```

Comment:
Why did you get rid of the finally block?

Reply:
because after you send that message it doesnt clear instantaneously, after i removed "finally" it did

Reply:
Sorry, what is "it"?

Reply:
when you click send, the message remains in the bar for a few seconds, after removing "finally" the message clears instantaneously. if you want me to revert it, i will.

Reply:
I'm not seeing that on my end. Could you try again with the finally block in there?

Reply:
We need that to enforce the clear just in case an error is thrown in the try block.